docs/semgrep-app/sso.md

```diff
@@ -16,19 +16,36 @@ OAuth2/OIDC is configured through the **Semgrep App Dashboard**.
 
 To set up SSO:
 
-1. From the **App Dashboard**, click on **Settings > Configure SSO tab**.
+1. From the **App Dashboard**, click on **Settings > Configure SSO: OpenID**.
 2. Copy the `Provider ID` and `Redirect URI`.
 ![Finding providerId and RedirectURL via the Semgrep App](../img/sso-finding-providerId-and-Redirect-URL.png "Finding Provider ID and RedirectURI via the Semgrep App")
 3.Generate a `Client ID` and `Client Secret` through your authentication provider by pasting `Provider ID` and `Redirect URI` values as needed.
 ![Generating Client ID and Client Secret via the Okta](../img/sso-clientID-clientSecret.png "Generating Client ID and Client Secret through Okta")
-4. From your authentication provider, copy the values for `Base URL/Domain` and `Email Domain` to Semgrep's Configure SSO tab. `Base URL/Domain` is `Okta domain` for Okta SSO.
+4. From your authentication provider, copy the values for `Base URL/Domain` and `Email Domain` to Semgrep's Configure SSO: OpenID tab. `Base URL/Domain` is `Okta domain` for Okta SSO.
 5. Provide a descriptive `Display Name`.
 ![Providing the Base URL/Domain, Display Name, and Email Domain](../img/sso-providing-BaseURL-DisplayName-EmailDomain.png "Providing the Base URL/Domain, Display Name, and Email Domain")
 
 In case you encounter issues during the setup process, please reach out to [support@r2c.dev](mailto:support@r2c.dev) for assistance.
 
 ## SAML 2.0
 
-SAML 2.0 currently requires manual support. Please reach out via Slack or [support@r2c.dev](mailto:support@r2c.dev).
+SAML2.0 is configured through the **Semgrep App Dashboard**. 
+
+To set up SSO:
+
+1. From your authentication provider, create the SAML app.
+![Creating SAML app through Okta](../img/saml-creating-app.png "Creating SAML app through Okta")
+2. From the **App Dashboard**, click on **Settings > Configure SSO: SAML**.
+3. Copy the `Provider ID`, the `Single sign on URL`, and `Audience URI` and paste the values as needed in your authentication provider.
+![Finding Provider ID, Single sign on URL, and Audience URI via the Semgrep App](../img/saml-finding-providerId-SsoURL-and-AudienceURI.png "Finding Provider ID, Single sign on URL, and Audience URI via the Semgrep App")
+4. From your authentication provider, add in 2 attribute statements `name` and `email`.
```

Comment:
```suggestion
4. From your authentication provider, add in two attribute statements `name` and `email`.
```
Tidbit about handling numbers for consistency everywhere in the docs: https://developers.google.com/style/numbers?hl=en